client/snap_op.go

```diff
@@ -175,8 +177,8 @@ func (client *Client) doSnapAction(actionName string, snapName string, options *
 }
 
 func (client *Client) doMultiSnapAction(actionName string, snaps []string, options *SnapOptions) (changeID string, err error) {
-	if options != nil {
-		return "", fmt.Errorf("cannot use options for multi-action") // (yet)
+	if options != nil && options.Dangerous {
+		return "", ErrDangerousNotApplicable
```

Comment:
this is wrong: before this change, no options were allowed. You now should be allowing only `Purge`. Instead this change lets you do `snap install --classic foo bar` which will not DTRT but will also not fail (AFAICT).

Reply:
in fact, checking "are any options other than Purge set" sounds quite hard, and I have zero problems with saying "purge only works for single-snap operations" in snap itself directly, and continuing to disallow options for multi

Reply:
Might be a good idea, thanks, will do that.